test/backend_fused_op.in.cpp

```diff
@@ -715,3 +715,49 @@ NGRAPH_TEST(${BACKEND_NAME}, mvn_mean_variance_normalization_split_channels)
 
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, grn_4d)
+{
+    Shape data_shape{1, 2, 3, 4};
```

Comment:
Not very important thing, but can you add `const` to everything you can in those tests?